src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -1166,6 +1184,30 @@ def db_update(
     if zone_redundant is not None:
         instance.zone_redundant = zone_redundant
 
+    #####
+    # Set other (serverless related) properties
+    #####
+    if min_capacity:
+        instance.min_capacity = min_capacity
+
+    if auto_pause_delay:
+        instance.auto_pause_delay = auto_pause_delay
+
+    if compute_model is not None:
+        # Determine server location
+        server_location = _get_server_location(
+            cmd.cli_ctx,
+            server_name=server_name,
+            resource_group_name=resource_group_name)
+
+        # reset sku name for update
+        instance.sku.name = ""
+
+        instance.sku = _find_db_sku_from_capabilities(cmd.cli_ctx, 
+            server_location, 
+            instance.sku, 
+            requesting_serverless= compute_model == "Serverless")
```

Comment:
Why is this a separate step from `_db_elastic_pool_update_sku` ?

Reply:
Yes, this should be part of the function. I have added it there.